### PR TITLE
Add DevDocs.app v0.1.0

### PR DESCRIPTION
**Pull request by MicTech**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

### CODE REVIEW

File: Casks/devdocs.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'devdocs' do
+  version '0.1.0'
+  sha256 'ede46d788775fbac7147dc45589f2de27762273962e4219730555896e00c576a'
+
+  url "https://github.com/egoist/devdocs-app/releases/download/v#{version}/DevDocs-#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/devdocs.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/devdocs.rb (hunk 1 above)
Comment (MicTech) [inline Casks/devdocs.rb]: @vitorgalvao I just fix that.

File: Casks/devdocs.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'devdocs' do
+  version '0.1.0'
+  sha256 'ede46d788775fbac7147dc45589f2de27762273962e4219730555896e00c576a'
+
+  url 'https://github.com/egoist/devdocs-app/releases/download/v0.1.0/DevDocs-0.1.0.dmg'
```
Comment (vitorgalvao) [inline Casks/devdocs.rb]: You reverted this line. Don’t.

File: Casks/devdocs.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  appcast 'https://github.com/egoist/devdocs-app/releases.atom'
```
Comment (vitorgalvao) [inline Casks/devdocs.rb]: This is incomplete. Please read the documentation I pointed you to, and don’t forget to rerun `audit` and `style` after changes.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/devdocs.rb (hunk 2 above)
Comment (MicTech) [inline Casks/devdocs.rb]: Sorry, my mistake. It should be ok now.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**